src/Illuminate/Database/Console/Seeds/SeedCommand.php

```diff
@@ -33,17 +34,25 @@ class SeedCommand extends Command
      */
     protected $resolver;
 
+    /**
+     * Timer instance.
+     *
+     * @var \Illuminate\Support\Stopwatch
+     */
+    protected $watch;
+
     /**
      * Create a new database seed command instance.
      *
-     * @param  \Illuminate\Database\ConnectionResolverInterface  $resolver
-     * @return void
```

Comment:
Please re-add the return annotation. It's the same across all other classes in Laravel.